nservicebus/msmq/distributor/deploying-to-a-cluster.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Deploying distributors to a cluster
+related:
+- nservicebus/msmq/troubleshooting
+reviewed: 2017-05-10
+---
```

Comment:
Should we also listen to the location of the previous documentation or is that this url?